app/presenters/flood_risk_engine/enrollment_presenter.rb

```diff
@@ -0,0 +1,22 @@
+#
+# A generalised presenter which adds helpers to simplify accessing enrollment data
+# from a view.
+#
+module FloodRiskEngine
+  class EnrollmentPresenter
+    delegate :exemption_location, to: :enrollment
+    delegate :grid_reference, to: :exemption_location, allow_nil: true
+
+    def initialize(enrollment)
+      @enrollment = enrollment
+    end
+
+    def organisation_type
+      enrollment.organisation && enrollment.organisation.org_type.to_sym
+    end
+
+    private
```

Comment:
@reggieb  My take on this is that a presenter should enforce the [law of demeter](http://devblog.avdi.org/2011/07/05/demeter-its-not-just-a-good-idea-its-the-law/); exposing the enrollment encourages train wrecks in the view eg. `@enrollment_presenter.enrollment.object_a.object_b.first.name`
So enrollment is private to try and discourage that from happening. Happy to discuss a better way to do this.